### ISSUE TITLE
508-defect-3 [COGNITION]: Links should be written and styled as links

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22730]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh Kim

## Details

"Request help with VA Form 5655" is _styled_ as a link but is written as a `button` which is materially dishonest. Users may become frustrated that they are unable to open a new tab. Screen reader users may be confused that the text seems more like a link despite being announced as a button.

For more on this long-tail effort, check out our [buttons vs. links Mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1610052905994/253723ef4b2ee9a726f538876a257df94d468db1) and this [Slack thread discussing _why_ material honesty is so important](https://dsva.slack.com/archives/C8E985R32/p1610659601009200?thread_ts=1610652897.002600&cid=C8E985R32).

"A few years at an accessibility event, a blind man was telling a story about how he was on the phone with someone who was trying to help him sign up for a service. The person on the phone told him to click the “sign up button” and he couldn’t find it because it was a link. He told the story with a lot more gravity than I can, but that was the point where I realized how important the human-human communication part of accessibility is, it’s not just machine to human." - @timwright12 

## Acceptance Criteria

- [ ] "Request help with VA Form 5655" is written with an `a` tag OR
- [ ] "Request help with VA Form 5655" is styled as a button

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Solution (if known)

Use `a` instead of `button` or style the `button` as a `button`

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [VA Design System guidance on buttons](https://design.va.gov/components/buttons#guidance)

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/14154792/113760040-c910dc00-96e3-11eb-9e52-36610af27bfe.png)

**Comment by denisecoveyduc** [comment]
@joshkimux @AngelaFowler82  I am doing a little GH clean up and wondered if you could just take a quick look at this issue for me to validate that this is still a concern? Since (and before I joined the team) the FSR has gone through many changes and this may or may not have already been addressed. Thanks in advance for checking it out for us.

**Comment by AngelaFowler82** [comment]
Request help with 5655 is still coded as a button. Since it takes the user to a new page, it should be coded and styled as a link, though it could be styled as an action link for visual emphasis.

**Comment by joshkimux** [comment]
<img width="1413" alt="Screen Shot 2022-01-12 at 10 27 56 AM" src="https://user-images.githubusercontent.com/14154792/149216653-17c5d121-80a6-4b79-b5aa-626e33fa2848.png">

Confirming what Angela caught here. In regards to whether or not this should be an action link may need a little more conversation with the designers.

This also may not be an issue if this dynamic wizard is replaced with a one thing per page based wizard or other content changes.

**Comment by denisecoveyduc** [comment]
Thank you both! @rileyorr drawing your attention to this one.. an old 508 defect picked up from our GH Clean up process.. thoughts?

**Comment by denisecoveyduc** [comment]
Shifting this over to our Tech debt epic to ensure we can address this and move it forward. It was originally captured as part of the staging review for the initial FSR launch.

**Comment by mgleo** [comment]
https://staging.va.gov/manage-va-debt/request-debt-help-form-5655/introduction
This is resolved as "Request help with VA Form 5655" is now styled as a button @HeatherRienks @Tom-Davis
